src/Engine/XYCoord.java

```diff
@@ -16,6 +16,15 @@ public boolean equals(int x, int y)
   {
     return x == xCoord && y == yCoord;
   }
+  
+  public int getDistance(XYCoord other)
+  {
+    return getDistance(other.xCoord, other.yCoord);
+  }
+  public int getDistance(int x, int y)
```

Comment:
I'm...  surprised we didn't already have this.

Reply:
Nope, but we do have `ManhattanDistanceComparator`

Reply:
Yeah. Useful, but not the same.